diagrams/aws/network.py

```diff
@@ -40,6 +40,14 @@ class GlobalAccelerator(_Network):
     _icon = "global-accelerator.png"
 
 
+class InternetGateway(_Network):
+    _icon = "internet-gateway.png"
+
+
+class NateGateway(_Network):
```

Comment:
NatGateway? It seems a typo. Please update the resource filename.

And `NAT` should be an uppercase, you can add it to upper words list: https://github.com/mingrammer/diagrams/blob/master/config.py#L50